### ISSUE TITLE
Converter paths error although paths are correct

### ISSUE THREAD

**Issue by sk365** [flingo64/PhotoStation-Upload-Lr-Plugin#83]
Hi, I cannot get past the Publish Manager Initial Screen ( German: "Veröffentlichungsmanager") because it tells me paths for converter tools were missing or wrong. However, paths are correct, files exist and are executable.  I am using a Mac M3 /OS 14.6.1/  LR 13.5 

Please help.

**Comment by flingo64** [comment]
Paths look good, can you please check if all four converter tools can be executed from terminal

**Comment by sk365** [comment]
Yes, I can execute all four converters from terminal. (When doing so, they come up with usage messages asking for arguments.)

**Comment by sk365** [comment]
Problem has been resolved. A path figured up twice in the entry; I failed to see it. Sorry.

**Comment by sk365** [comment]
Problem has been resolved. A path figured up twice in the entry; I failed to see it. Sorry.
